packages/coil-extension/src/popup/contants.ts

```diff
@@ -1,9 +1,10 @@
 export enum ROUTES {
   streaming = 'streaming',
-  streamingNoWebMo = 'streaming/notmonetized',
-  streamingNoMembership = 'streaming/nomembership',
-  streamingCoil = 'streaming/coil',
-  streamingCoilDiscover = 'streaming/coil/discover',
+  // streamingWebMo = 'streaming/monetized',
+  // streamingNoWebMo = 'streaming/notmonetized',
+  // streamingNoMembership = 'streaming/nomembership',
+  // streamingCoil = 'streaming/coil',
+  // streamingCoilDiscover = 'streaming/coil/discover',
```

Comment:
Did you mean to keep these commented out?

Reply:
Nope, and I just realized the file is misspelled